vendors/nordic/boards/nrf52840-dk/ports/ble/iot_ble_hal_gap.c

```diff
@@ -823,7 +823,10 @@ ret_code_t prvBTAdvDataConvert( ble_advdata_t * xAdvData,
         xAdvData->p_slave_conn_int->max_conn_interval = pxParams->ulMaxInterval;
         xAdvData->p_slave_conn_int->min_conn_interval = pxParams->ulMinInterval;
 
-        if( ( pcManufacturerData != NULL ) && ( usManufacturerLen != 0 ) )
+        uint8_t company_id_size = sizeof( uint16_t );
```

Comment:
Variable names follows hungarian notations. uint8_t variables prefix with "uc" followed by camel case. For example ucCompanyIDSize. uint16_t prefix with "us"

Reply:
Also could we define a constant for this ?

Reply:
Guidelines at following link does not mention about hungarian notation:
https://github.com/aws/amazon-freertos/blob/master/CONTRIBUTING.md#coding-style

I have defined a constant for company ID size.